cli/preprocess-xgettextjs-match.js

```diff
@@ -121,7 +111,7 @@ function makeDoubleQuoted( literal ) {
 
 	// ES6 string
 	if ( literal.charAt( 0 ) === '`' ) {
-		return '"' + literal.substring( 1, literal.length - 1 ).replace( /`/g, '\`' ).replace( /(\\|")/g, '\\$1' ) + '"';
+		return '"' + literal.substring( 1, literal.length - 1 ).replace( /`/g, '\`' ).replace( /(\\|")/g, '\\$1' ).replace( /\n/g, ' ' ) + '"';
```

Comment:
I'm a bit uncomfortable with the way the `replace( /\n/g, ' ' )` introduces a break between what the developer sees and what translators see.

I don't see any translations right now that would go wrong right now ( https://translate.wordpress.com/projects/wpcom/fr/default/?filters%5Bterm%5D=%5Cn&filters%5Buser_login%5D=&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bview%5D=&filter=Filter&sort%5Bby%5D=priority&sort%5Bhow%5D=desc ), but it would be really hard to spot if something did come up.

If you were really keen to do this, we should add a complementary change to our runtime  `translate()` logic, but I think we should just use the linting rule.